### DISCUSSION TITLE
Écart entre le rôle et les caractéristiques

### DISCUSSION THREAD

**Discussion by vouiv-re** [fibreville/minotaure_php#85] [Boite à idées]
Petite remarque, qui m'a plus fait marrer que ça ne m'a gêné : 

Ce soir en tant que dragon je faisais beaucoup de test d'esprit, mais j'avais choisi les caractéristiques maximales en corps, ce qui est un poil handicapant.

Est-ce qu'on assume ce côté en peu chaotique, qui permet de créer de la variété et des personnages qui ne sont pas des gros Bill ? C' est pas forcément cohérent, et ça peut éventuellement frustrer un joueur surtout si on le sélectionne et on le sanctionne pour un trait qu'il n'a pas choisit.

**Comment by uurshin** [comment]
Ce n'est pas vraiment assumé, c'est plus un héritage du système de base.
J'ai essayé de compenser en attribuant du loot en fonction de l'espèce, qui était censé booster les caracs naturelles, mais bon ce n'est pas viable car il faut le faire dès qu'un nouveau personnage est crée.

Perso ça me dérange pas trop que les caracs ne soient pas 100% alignées avec le personnage car dans la vie il y a des disparités :D.
Plus sérieusement, je pense qu'on pourrait faire en sorte que les tags aient des propriétés associés, donc de pouvoir éditer un tag pour lui associer des bonus et des malus. Ces modificateurs seraient ajoutées automatiquement aux persos avec le tag, et enlevées si le tag est supprimé.
